k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -3022,7 +2995,8 @@ public void onMessageBuildCancel() {
     @Override
     public void onMessageBuildException(MessagingException me) {
         Log.e(K9.LOG_TAG, "Error sending message", me);
-        Toast.makeText(MessageCompose.this, getString(R.string.send_aborted), Toast.LENGTH_LONG).show();
+        Toast.makeText(MessageCompose.this,
+                getString(R.string.send_failed_reason, me.getLocalizedMessage()), Toast.LENGTH_LONG).show();
         currentMessageBuilder = null;
```

Comment:
I figured. But what we really want is an error code -> error message mapping inside K-9 Mail. That way we can provide appropriate error messages that can be localized by our translators.